imap/ctl_conversationsdb.c

```diff
@@ -501,6 +501,29 @@ static int next_diffable_record(struct cursor *c)
     }
 }
 
+static void printer(const char *type, struct cursor *c)
+{
+    if (!verbose) return;
+    static struct buf keybuf = BUF_INITIALIZER;
+    static struct buf databuf = BUF_INITIALIZER;
+    buf_reset(&keybuf);
+    buf_reset(&databuf);
+    size_t i;
+    for (i = 0; i < c->keylen; i++) {
+        if (c->key[i] > 31 && c->data[i] < 127)
```

Comment:
There's a bug here, cf `c->key[i]` in the first term and `c->data[i]` in the second

Also, I think this could just be `if (isprint(c->key[i]))` using isprint() from ctype.h anyway

Reply:
Ouch, yeah that's a copy-paster